supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystem.scala

```diff
@@ -122,6 +207,7 @@ trait SharedFileSystem extends PathFactory {
     // If localizing for a docker job, remove soft-link as an option
```

Comment:
Please update the comment to describe the action being taken here for `cached-copy` as well.